### PR TITLE
Flake8 fixes for the utils

### PR DESCRIPTION
**Pull request by parag-may4**
Flake8 fixes for the utils of central sdk.
It contains fixes which are flagged by flake8 and not other functionality is impacted. 
Note: For tech-support file, PR is raised by vvb already, so other files are taken care by this PR.

### CODE REVIEW

File: ucscentralsdk/utils/ccoimage.py [hunk 1]
```diff
@@ -105,6 +106,9 @@ def get_ucscentral_cco_image_list(username=None, password=None, mdf_id_list=None
     import xml.dom
     import xml.dom.minidom
     import base64
+    import sys
+    if sys.version[0] == '3':
+        raw_input = input
```
Comment (vvb) [inline ucscentralsdk/utils/ccoimage.py]: @parag-may4 should be
```
try:
    import raw_input
except:
    import input
```

File: ucscentralsdk/utils/ccoimage.py [hunk 2]
```diff
@@ -195,7 +199,8 @@ def get_ucscentral_cco_image_list(username=None, password=None, mdf_id_list=None
                     image.checksum_md5 = property_node.getAttribute("value")
                     continue
             if property_node.getAttribute(
-                    "name") == _UcscentralCcoImageList.IDAC_TAG_FILE_DESCRIPTION:
+                "name") == \
+                    _UcscentralCcoImageList.IDAC_TAG_FILE_DESCRIPTION:
```
Comment (vvb) [inline ucscentralsdk/utils/ccoimage.py]: reformat!

File: ucscentralsdk/utils/ccoimage.py [hunk 3]
```diff
@@ -217,16 +222,18 @@ def get_ucscentral_cco_image(image, file_dir, proxy=None):
...
     Example:
         image_list = get_ucscentral_cco_image_list("username", "password")\n
-        get_ucscentral_cco_image(image=image_list[0], file_dir="/home/user/images")\n
+        get_ucscentral_cco_image(image=image_list[0],
+                                file_dir="/home/user/images")\n
     """
 
     if not image:
-        raise UcsCentralValidationException("Provide an image to be downloaded.")
+        raise UcsCentralValidationException("Provide an image for downloaded")
```
Comment (vvb) [inline ucscentralsdk/utils/ccoimage.py]: grammar. keep the original string and reformat to 2 lines.

File: ucscentralsdk/utils/ccoimage.py [hunk 4]
```diff
@@ -217,16 +222,18 @@ def get_ucscentral_cco_image(image, file_dir, proxy=None):
...
     if not isinstance(image, UcscentralCcoImage):
-        raise UcsCentralValidationException("Object is not of type UcscentralCcoImage")
+        raise UcsCentralValidationException("Object is not of "
+                                            "type UcscentralCcoImage")
 
     if not file_dir:
-        raise UcsCentralValidationException("Provide file_dir to download image to.")
+        raise UcsCentralValidationException("Provide file_dir for download")
```
Comment (vvb) [inline ucscentralsdk/utils/ccoimage.py]: same as above..

File: ucscentralsdk/utils/ucscentralbackup.py [hunk 5]
```diff
@@ -131,7 +135,8 @@ def backup_ucscentral(handle, file_dir, file_name, timeout_in_sec=600,
         if duration == 0:
             handle.remove_mo(mgmt_backup)
             handle.commit()
-            raise UcsCentralOperationError("Backup ucscentral", " operation timed out")
+            raise UcsCentralOperationError(
+                "Backup ucscentral", " operation timed out")
```
Comment (vvb) [inline ucscentralsdk/utils/ucscentralbackup.py]: ucscentral -> UcsCentral

File: ucscentralsdk/utils/ccoimage.py (hunk 1 above)
Comment (parag-may4) [inline ucscentralsdk/utils/ccoimage.py]: The best way I found to change this is :   
```
try:                                                                           
    input = raw_input                                                      
except NameError:                                                              
    pass  

username = input("Username: ")     

`````

**Pull request merged**